bodhi/server/validators.py

```diff
@@ -800,48 +778,60 @@ def validate_update(request, **kwargs):
         request.errors.status = HTTPNotFound.code
 
 
-def validate_update_owner(request, **kwargs):
+def ensure_user_exists(param, request):
     """
-    Make sure the referenced update owner is an existing user.
+    Ensure the user referenced by param exists and if it does replace it with the User object
```

Comment:
The test failure wants you to end this sentence with a period:

```
bodhi/server/validators.py:807 in public function `ensure_user_exists`:
        D400: First line should end with a period (not 't')
```

You can manually run pydocstyle on the code to make sure it passes with tox, or by running this:

```
$ pydocstyle  --add-ignore D413 bodhi/server/validators.py
```

Reply:
Added period.